src/pages/trader-tools/margin-calculator/_margin-calculator.js

```diff
@@ -395,75 +428,71 @@ const MarginCalculator = () => {
-                            )}
-                        </Text>
-                    </TextWrapper>
-                </RightContent>
-            </ContentContainer>
-
-            <BottomContent direction="column">
-                <BottomText mb="2.4rem">
-                    <Localize
-                        translate_text="To view the asset price, go to Deriv MetaTrader 5 (DMT5), click on the <0>View </0> tab and select<0> Market Watch</0>, then right-click on the symbol you want to trade and select <0>Specification.</0>"
-                        components={[<strong key={0} />]}
-                    />
-                </BottomText>
-
-                <LinkWrapper>
-                    <StyledLinkButton
-                        tertiary="true"
-                        to="https://app.deriv.com/mt5"
-                        external="true"
-                        target="_blank"
-                    >
-                        {localize('Go to DMT5 dashboard')}
-                    </StyledLinkButton>
-                    <StyledLinkButton secondary="true" to="/trade-types/margin">
-                        {localize('Learn more about margin')}
-                    </StyledLinkButton>
-                </LinkWrapper>
-            </BottomContent>
-        </StyledSection>
+                        <Accordion has_single_state>
+                            <AccordionItem
+                                header={localize('Margin required')}
+                                header_style={header_style}
+                                style={item_style}
+                                plus
+                            >
+                                <Text mb="16px">
+                                    {localize(
+                                        'Let’s say you want to trade two lots of EUR/USD with an asset price of 1.10 USD and leverage of 100.',
+                                    )}
+                                </Text>
+
+                                <Show.Desktop>
+                                    <QueryImage data={data.margin_formula} alt={'Margin formula'} />
+                                </Show.Desktop>
+
+                                <Show.Mobile>
+                                    <QueryImage
+                                        data={data.margin_formula_mobile}
+                                        alt={'Margin formula mobile'}
+                                    />
+                                </Show.Mobile>
+
+                                <FormulaText>
+                                    <StyledOl>
+                                        <li>
+                                            <span>
+                                                <Localize translate_text=" One standard lot of forex = 100,000 units" />
```

Comment:
is it intentional to have space in the beginning?

Reply:
can we have margin instead?

Reply:
Will revise it based on the new design.